cpp/tests/utilities/column_utilities.cu

```diff
@@ -124,6 +182,21 @@ void expect_columns_equal(cudf::column_view const& lhs, cudf::column_view const&
   }
 }
 
+} // namespace anonymous
+
+void expect_columns_equal(cudf::column_view const& lhs, cudf::column_view const& rhs,
+                          bool print_all_differences) 
+{
+  column_comparison<corresponding_rows_unequal>(lhs, rhs, print_all_differences);
+}
+
+void expect_columns_equivalent(cudf::column_view const& lhs,
+                               cudf::column_view const& rhs,
```

Comment:
This function should also differ in the `expect_column_properties_equal`. I.e., `expect_columns_equal` requires both columns to have identical nullability, even if the null count is zero.